support-workers/src/main/scala/com/gu/services/Services.scala

```diff
@@ -41,5 +42,6 @@ class Services(isTestUser: Boolean, val config: Configuration) {
   lazy val redemptionService = RedemptionTable.forEnvAsync(TouchPointEnvironments.fromStage(stage, isTestUser))
   lazy val catalogService = CatalogService(TouchPointEnvironments.fromStage(stage, isTestUser))
   lazy val giftCodeGenerator = new GiftCodeGeneratorService
+  lazy val bigQueryService = new BigQueryService(bigQueryConfigProvider.get(isTestUser))
```

Comment:
we do have a lot in this Services class, it's reminding me strongly of the TouchPointBackend class now! 😢

Reply:
Yeah it is quite big, could do with a refactor at some point.